posthog/hogql/test/test_parser_cpp.py

```diff
@@ -2,4 +2,7 @@
 
 
 class TestParserCpp(parser_test_factory("cpp")):
-    pass
+    def test_empty(self):
+        # this test only exists to make pycharm recognise this class as a test class
+        # the actual tests are in the parent class
+        pass
```

Comment:
Oh interesting, I don't think this was needed before, so curious how you stumbled upon the need

Reply:
Nothing major, I just wanted to get the green arrow to appear in the UI, when I was trying to figure out when these tests were broken on main